io.github.Soundux.yml

```diff
@@ -0,0 +1,26 @@
+app-id: io.github.Soundux
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: soundux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --filesystem=home
+  - --filesystem=host:ro
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: soundux
+    buildsystem: cmake
```

Comment:
If it works then switching this to cmake-ninja saves a bunch of build time but it's just a nitpick and entirely your choice.

Reply:
Thanks, it seems to work so I updated it